app/services/commitments/minimum/calculate_true_up_fee_service.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module Commitments
+  module Minimum
+    class CalculateTrueUpFeeService < BaseService
+      def initialize(invoice_subscription:)
+        @invoice_subscription = invoice_subscription
+        @minimum_commitment = invoice_subscription.subscription.plan.minimum_commitment
+
+        super
+      end
+
+      def call
+        result.amount_cents = amount_cents
+        result
+      end
+
+      private
+
+      attr_reader :minimum_commitment, :invoice_subscription
+
+      delegate :subscription, to: :invoice_subscription
+
+      def amount_cents
+        return 0 if !minimum_commitment || fees_total_amount_cents >= commitment_amount_cents
+
+        commitment_amount_cents - fees_total_amount_cents
+      end
+
+      def commitment_amount_cents
+        result = Commitments::CalculateAmountService.call(
+          commitment: minimum_commitment,
+          invoice_subscription:,
+        )
+
+        result.commitment_amount_cents
+      end
+
+      def fees_total_amount_cents
+        helper_service = Commitments::HelperService.new(commitment: minimum_commitment, invoice_subscription:, current_usage: true)
```

Comment:
Layout/LineLength: Line is too long. [131/120] (https://rubystyle.guide#max-line-length)